testgrid/conformance/README.md

```diff
@@ -39,7 +39,7 @@ For running the conformance tests and obtaining the result files (`b)` and `c)`)
    ```sh
    git clone https://github.com/kubernetes/kubernetes.git && cd kubernetes && git checkout release-1.11
    ```
-   - run `make WHAT=test/e2e/e2e.test && make WHAT=ginkgo && make WHAT=cmd/kubectl` to build the test binaries
+   - run `make all WHAT="test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo cmd/kubectl"` to build the test binaries
```

Comment:
you can see this in the `kind-e2e.sh` script. I've been meaning to make this change. this is faster as it doesn't involve repeatedly doing all of the docker build setup